modules/installation-guide/partials/assembly_mounting-secrets-or-a-configmap-as-file-or-env-variable-in-container.adoc

```diff
@@ -0,0 +1,28 @@
+:parent-context-of-mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-container: {context}
+
+[id="mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-container_{context}"]
+= Mounting a Secret or a ConfigMap as a file or an environment variable into a {prod-short} container
+
+:context: mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-eclipse-che-container
+
+Secrets are {platforms-name} objects that store sensitive data such as:
+
+* usernames
+* passwords
+* authentication tokens
+* and configurations
+
+in an encrypted form.
+
+Users can mount a {platforms-name} Secret that contains sensitive data or a ConfigMap in a {prod-short} container as:
```

Comment:
![Screenshot_20210723_094351](https://user-images.githubusercontent.com/5887312/126746203-655832f7-9f4d-4425-bb80-049e0b6a9f50.png)

I'm not sure that it's correct to use `in a Che container` since it may be postgres, keycloak which are Che Cluster related but they are not really Che containers.

Maybe a bit better would be `in Che related containers`
Maybe a bit better would be `in Che managed containers`
But the above are not ideal as well.